tests/gtest/test_netif.cpp

```diff
@@ -464,6 +465,14 @@ TEST(Netif, WpanIfStateChangesCorrectly_AfterSettingNetifState)
     netif.Deinit();
 }
 
+void receiveTask(int aSockFd, uint8_t *aRecvBuf, struct sockaddr_in6 *aListenAddr)
```

Comment:
I think sleep after bind is simpler than opening a thread.

Reply:
As discussed, we used the thread solution for now. We will further investigate the underlying mechanisum later.